bin/uploadBrowser.js

```diff
@@ -72,7 +72,7 @@ var OS_IDENTIFIER = 2
 // Recipe tuples containing local relative paths to files, key locations on S3, and an os identifier
 var recipes = [
   // Linux
-  ['dist/Brave.tar.bz2', 'multi-channel/releases/CHANNEL/VERSION/linux64', 'linux'],
+  ['dist/Brave{{channelName}}.tar.bz2', 'multi-channel/releases/CHANNEL/VERSION/linux64', 'linux'],
```

Comment:
This is one I missed last time... not important though, since Linux users can't upgrade without apt/yum/dnf